src/main/java/com/commercetools/sync/inventories/SkuKeyTuple.java

```diff
@@ -0,0 +1,116 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+/**
+ * Helper class that holds sku and supply channel key - values used for {@link io.sphere.sdk.inventory.InventoryEntry}
+ * distinction. Presence of sku and key in created instance is similar as for inventory entry resource: supply channel
+ * key can be {@code null} value since it is optional in inventory entry, but sku has to be not null neither not empty.
+ * Can be used as key in Maps.
+ */
+final class SkuKeyTuple {
+
+    static final String SKU_NOT_SET_MESSAGE = "Sku is not set";
```

Comment:
Actually the end user of sync would never see that message as I filter out entries without SKU at the beginning of sync process. It's more like for someone who would reuse that class in a future. 5ccb33a58e92d07db435e6f2d57145311a058ffe